docs/my-website/docs/completion/function_call.md

```diff
@@ -103,11 +103,25 @@ messages = [
     {"role": "user", "content": "What is the weather like in Boston?"}
 ]
 
-def get_current_weather(location):
-  if location == "Boston, MA":
-    return "The weather is 12F"
+def get_current_weather(location: str, unit: str):
+    """Get the current weather in a given location
 
-functions = litellm.utils.function_to_dict(get_current_weather)
```

Comment:
Calling `function_to_dict` on `get_current_weather` results in a NoneType exception; A docstring must be present.